tools/pipelines/test-azure-frs.yml

```diff
@@ -25,8 +25,10 @@ pr: none
 
 resources:
   pipelines:
-  - pipeline: client   # Name of the pipeline resource
+  - pipeline: client
```

Comment:
@skylerjokiel @tylerbutler this is related to our standup discussion. We may not need any references to client pipelines here. Given that azure is its own monorepo, expectation is for azure-client to depend only on officially published releases/pre-preleases?